.github/workflows/maven-deploy-test-server.yml

```diff
@@ -10,13 +10,13 @@ on:
 jobs:
   maven-deploy-manual:
     if: ${{ github.event_name == 'workflow_dispatch' }}
-    name: Manual deploy
-    uses: wultra/wultra-infrastructure/.github/workflows/maven-deploy.yml@develop
+    name: Manual deploy test-server
+    uses: wultra/wultra-infrastructure/.github/workflows/maven-deploy-test-server.yml@develop
```

Comment:
Are you sure that this shared action exists?

Reply:
Oops, renaming the file in the IDE has some consequences.